app/models/episode.rb

```diff
@@ -310,9 +310,20 @@ def set_external_keyword
     identifiers = []
     [:published_at, :guid].each do |attr|
       # Adzerk does not allow commas or colons in keywords; omitting dashes for space
-      identifiers << self.send(attr).to_s.slice(0, 10).gsub(/[:,-]/,'')
+      identifiers << self.send(attr)
+                         .to_s
+                         .downcase
+                         .gsub(/[:,-]/,'')
+                         .gsub(/\s+/,'')
+                         .strip
+                         .slice(0, 10)
     end
-    identifiers << (title || 'undefined').slice(0, 20)
+    identifiers << (title || 'undefined').to_s
+                                         .downcase
+                                         .gsub(/[:,-]/,' ')
+                                         .gsub(/\s+/,' ')
```

Comment:
took me a second to notice the difference - this makes sense to me to keep spaces in the title, but not in the date and guid above (which shouldn't have space anyway!)